gpustat/core.py

```diff
@@ -337,6 +341,27 @@ def jsonify(self):
         return o
 
 
+class LostGPU(GPUStat):
+
+    def __init__(self, gpu_index, ex):
+        super().__init__(dict(
+            index=gpu_index,
+            name='((GPU is Lost))',
+            processes=None
+        ))
+        self.entry = self.NullEntry(self.entry)
+        self._gpu_index = gpu_index
+        self._ex = ex
+
+    class NullEntry:
```

Comment:
You can use the builtin defaultdict

Reply:
I wanted to make the entry read-only. Otherwise the accessed properties will be exposed in json (which is also fine though)

Reply:
It's better to have the same JSON structure for both available and unavailable GPUs.